helix-core/pom.xml

```diff
@@ -168,7 +168,7 @@ under the License.
     <resources>
       <resource>
         <directory>${basedir}/src/main/resources</directory>
-        <filtering>true</filtering>
+        <filtering>false</filtering>
```

Comment:
Why we change this?

Reply:
This caused maven to give a warning when building helix-core